source/_components/notify.xmpp.markdown

```diff
@@ -34,6 +34,7 @@ Configuration variables:
 - **password** (*Required*): The password for your given Jabber account.
 - **recipient** (*Required*): The Jabber ID (JID) that will receive the messages.
 - **tls** (*Optional*): Allow to disable TLS. Defaults to `true`.
+- **verify** (*Optional*): Allow to disable SSL certificate validity check (eg. self-signed certificate). Defaults to `true`.
```

Comment:
**Spelling & Grammar**:
`Allow to disable SSL certificate validity check (eg. self-signed certificate).` -> `Allow disabling SSL certificate validity check (e.g., self-signed certificate).`